### PR TITLE
Modules: keep custom *.config files also after `nf-core modules/subworkflows update`

### LINKED ISSUE #2658: nf-core modules/subworkflows update should not delete modules specific config files
### Description of feature

ie we have this file in fetchngs https://github.com/nf-core/fetchngs/blob/dev/modules/nf-core/sratools/fasterqdump/nextflow.config
And it gets deleted everytime, it's a bit of a hassle it would be nice to have an option or a warning or a mention or something

### PR DESCRIPTION
**Pull request by mashehu**
Closes #2658

### CODE REVIEW

**Comment by mashehu** [comment]
@nf-core-bot changelog

File: nf_core/components/update.py [hunk 1]
```diff
@@ -736,18 +736,29 @@ def move_files_from_tmp_dir(self, component, install_folder, repo_path, new_vers
...
             new_version (str): The version of the module/subworkflow that was installed.
         """
-        temp_component_dir = os.path.join(install_folder, component)
-        files = os.listdir(temp_component_dir)
-        pipeline_path = os.path.join(self.dir, self.component_type, repo_path, component)
+        temp_component_dir = Path(install_folder, component)
+        files = [f.name for f in temp_component_dir.iterdir() if f.is_file()]
+        pipeline_path = Path(self.dir, self.component_type, repo_path, component)
+
+        # check if both directories have the same file and warn if the compnent_dir has more files
```
Comment (mirpedrol) [inline nf_core/components/update.py:743]: the comment seems to be describing something else

**Review by mirpedrol — APPROVED** [review]
LGTM!

**Comment by mashehu** [comment]
@nf-core-bot changelog

**Pull request merged**